litmus-portal/graphql-server/pkg/database/mongodb/usermanagement/schema.go

```diff
@@ -14,7 +14,7 @@ type User struct {
 	State           *string `bson:"state"`
```

Comment:
What is the use of this State? Why is this not being updated for deactivate user call?

Reply:
In the code I don't see it being used anywhere, either we can remove it or use it

Reply:
I'll remove it then